solrcloudpy/connection.py

```diff
@@ -58,7 +58,8 @@ def __init__(self, server="localhost:8983",
                  timeout=10,
                  webappdir='solr',
                  version='5.3.0',
-                 request_retries=1):
+                 request_retries=1,
+                 use_https=False):
```

Comment:
So the default behavior is not to use HTTPS, and we have to enable it here?

Would it make sense to trigger this from an environment variable?

Reply:
Yes, when you create a SolrConnection you might want to indicate that the server(s) you are connecting to use HTTPS or HTTP, I can think of a scenario where I might want to have two different SolrConnections: one connecting to http://solr1:8983 and the other one to https://solr2:8983, if we set an environment variable for the HTTPS usage, this scenario would not be possible